content/v2.0/visualize-data/visualization-types/graph-single-stat.md

```diff
@@ -38,8 +38,8 @@ For consistent Single Stat results, the query should return a single table.
 {{% /note %}}
 
 ## Graph + Single Stat Controls
-To view **Graph + Single Stat** controls, click the settings icon ({{< icon "gear" >}})
-next to the visualization dropdown in the upper right.
+To view **Graph + Single Stat** controls, click **{{< icon "gear" >}} Customize** next to
```

Comment:
to be sure, for all these different controls, folks click the icon gear and Customize and get different visualizations?

Reply:
This is the button in the UI that users click to open visualization options:

![image](https://user-images.githubusercontent.com/181484/77664157-8c909900-6f43-11ea-9353-e360dc3ac763.png)